app/controllers/checkout_controller.rb

```diff
@@ -248,4 +249,11 @@ def rescue_from_spree_gateway_error(error)
   def permitted_params
     PermittedAttributes::Checkout.new(params).call
   end
+
+  def xhr_flash_errors
```

Comment:
discard_flash_errors is a better description of what the method does I think.

Reply:
Done.